db/dbService.ts

```diff
@@ -0,0 +1,45 @@
+const { Sequelize, Model, DataTypes } = require("sequelize");
+const { dbCredsForSequelize } = require('./config/config.ts');
+
+const models = require("./models");
+const dataBase = new Sequelize(dbCredsForSequelize);
+
+const pingDb = async () => {
+  try {
+    await dataBase.authenticate();
+    return "DB Auth successful";
+  } catch (err) {
+    return `DB Auth failed: ${err.toString()}`;
+  }
+};
+
+const insertUser = async (username, email, password) => {
+  try {
+    return await models.User.create({
+      username: username,
+      email: email,
+      password: password,
+    });
+  } catch (err) {
+    return `Failed to retreive users: ${err.toString()}`;
+  }
+};
+
+const getAllUsers = async () => {
```

Comment:
Just dropping a note here for us to utilize TS and add expected return types / etc. in the next PR - would be esp . useful for these